### PR TITLE
Fixed signing remove repo argument 

### PR DESCRIPTION
**Pull request by midnightercz**
- Replaced quay_org passed from cli with quay_namespace used in taret settings
- set quay token from env
- set static quay host

### CODE REVIEW

File: pubtools/_quay/iib_operations.py [hunk 1]
```diff
@@ -136,6 +136,7 @@ def _remove_index_image_signatures(
         current_signatures (list): List of current signatures.
         target_settings (dict): Target settings.
     """
+    set_aws_kms_environment_variables(target_settings, "cosign_signer")
```
Comment (querti) [inline pubtools/_quay/iib_operations.py:139]: Is this a part of the PR?

File: pubtools/_quay/remove_repo.py [hunk 2]
```diff
@@ -135,9 +136,18 @@ def remove_repositories(repositories: str, settings: dict[str, Any]) -> None:
             continue
         outdated_manifests.append((mad.digest, tag, repo))
 
+    signer_settings = {}
+    for k, v in settings.items():
```
Comment (querti) [inline pubtools/_quay/remove_repo.py:140]: Maybe you can just use deepcopy instead of iterating it?

File: pubtools/_quay/remove_repo.py [hunk 3]
```diff
@@ -135,9 +136,18 @@ def remove_repositories(repositories: str, settings: dict[str, Any]) -> None:
...
+        if k == "quay_org":
+            signer_settings["quay_namespace"] = v
```
Comment (querti) [inline pubtools/_quay/remove_repo.py:142]: Can you give an explanation in the commit message of this change and why it was necessary?

Comment (midnightercz) [reply pubtools/_quay/remove_repo.py:142]: sure

File: pubtools/_quay/iib_operations.py (hunk 1 above)
Comment (midnightercz) [reply pubtools/_quay/iib_operations.py:139]: no, that is left over after rebase

File: pubtools/_quay/tag_docker.py [hunk 4]
```diff
@@ -743,6 +743,7 @@ def untag_image(self, push_item: Any, tag: str) -> None:
 
         cert, key = get_pyxis_ssl_paths(self.target_settings)
 
+        set_aws_kms_environment_variables(self.target_settings, "cosign_signer")
```
Comment (querti) [inline pubtools/_quay/tag_docker.py:746]: Should this be a part of the PR?

**Review by emilyzheng — APPROVED** [review]

**Review by querti — APPROVED** [review]

**Pull request merged**